src/modules/data.js

```diff
@@ -0,0 +1,5 @@
+const data = {
+  toDoArr: [],
+};
+
+export default data;
```

Comment:
please consider not using global variables in your project. it could be easily overwrite or changed in the values somewhere else which make it hard to debug.